instrumentation/log4j/log4j-mdc-1.2/javaagent/build.gradle.kts

```diff
@@ -8,7 +8,8 @@ muzzle {
     module.set("log4j")
     versions.set("[1.2,)")
     // version 1.2.15 has a bad dependency on javax.jms:jms:1.1 which was released as pom only
-    skip("1.2.15")
+    skip("1.1.3", "1.2.15")
```

Comment:
if you really want to add this then I think you should also add a comment why you are skipping this version

Reply:
> if you really want to add this then I think you should also add a comment why you are skipping this version

I added `classLoaderMatcher` to solve the problem.